Docker/launcher_entrypoint.sh

```diff
@@ -16,7 +16,8 @@ smart_run () {
 
 # Create FIFO for emitter
 # https://github.com/screwdriver-cd/screwdriver/issues/979
-smart_run mkfifo -m 666 /opt/sd/emitter
+smart_run mkdir -p /sd
+smart_run mkfifo -m 666 /sd/emitter
```

Comment:
is the only dynamically created file in the whole ecosystem which needs to be written into ?

Reply:
yes, other files are mounted in `/opt/sd`